### PR TITLE
Feature/milestone 4

### PR DESCRIPTION
**Pull request by oscardelalanza**
#### What was made

- `calculate` and `operate` files added to the project

- `operate` module implemented to make the arithmetic operations with the help of `Big.js`

- `calculate` module implemented to handle a calculator object

- `calculate` module was imported to `App` component

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import operate from 'operate';
```
Comment (AdnanAfsari) [inline src/logic/calculate.js]: `./` add this, throwing error because of this.
imports from the same folder should starts with './'

**Review by AdnanAfsari — CHANGES_REQUESTED** [review]
**Status: Changes Required**

Hello @oscardelalanza ✋ 

Nice job, everything looks good but a minor error to fix to get approval. Please check my comment and fix the issue then submit for another review.

Happy Coding
Adnan Afsari

File: src/logic/calculate.js (hunk 1 above)
Comment (oscardelalanza) [reply src/logic/calculate.js]: really? you could have approved the project and ask for that insignificant change...

**Review by fatymahmed — APPROVED** [review]
## Hi @oscardelalanza,
**Great job making the change 🎉🎉**
## ✔️ Approved!

**Pull request merged**